src/python/tensorflow_cloud/core/validate.py

```diff
@@ -22,6 +22,7 @@
 
 from . import machine_config
 from . import gcp
+from . import deploy
```

Comment:
We should not import deploy module here. Please import TF version here instead.

Reply:
Done. Perhaps in a follow up PR it might be a good idea to factor out the version check to a utility module.